lib/samples/apply_simple_renderer_to_feature_layer/apply_simple_renderer_to_feature_layer_sample.dart

```diff
@@ -62,20 +63,17 @@ class _ApplySimpleRendererToFeatureLayerSampleState
                 controllerProvider: () => _mapViewController,
```

Comment:
Another little tidbit -- we want to add `top: false` to SafeAreas when the SafeArea has a map view at the top. (When the sample is launched individually, there is no AppBar, so we want the map to run all the way up under the notch and what-not.)